lib/dnsimple/client/registrar.rb

```diff
@@ -22,6 +22,26 @@ def check_domain(account_id, domain_name, options = {})
         Dnsimple::Response.new(response, Struct::DomainCheck.new(response["data"]))
       end
 
+      # Checks the premium price for a domain
+      #
+      # @see https://developer.dnsimple.com/v2/registrar/#premium-price
+      #
+      # @example Check whether example.com is available:
+      #   client.registrar.get_domain_premium_price(1010, "ruby.codes")
+      #
+      # @param  [Integer] account_id the account ID
+      # @param  [#to_s] domain_name the domain name to check
+      # @param  [Hash] options
+      # @return [Struct::DomainPremiumPrice]
+      #
+      # @raise  [RequestError] When the request fails.
+      def get_domain_premium_price(account_id, domain_name, options = {})
```

Comment:
Given in Ruby we use `domain` instead of `get_domain`, I suggest to use `domain_premium_price`. What do you think?